app/code/core/Mage/Adminhtml/Block/Cache/Grid.php

```diff
@@ -96,6 +96,15 @@ protected function _prepareColumns()
             'frame_callback' => [$this, 'decorateStatus']
         ]);
 
+        $this->addColumn('refreshed_at', array(
+            'header'    => $this->__('Refreshed At'),
+            'align'     => 'left',
+            'index'     => 'refreshed_at',
+            'type'      => 'datetime',
+            'width'     => '180',
+            'sortable'  => false,
+        ));
```

Comment:
Short array syntax and use existing text in `Mage_Adminhtml.csv`:
https://github.com/OpenMage/magento-lts/blob/e2e738a884f24e09e05e7ad098946ac4d7ff877d/app/locale/en_US/Mage_Adminhtml.csv#L732-L732
```suggestion
        $this->addColumn('performed_at', [
            'header'    => $this->__('Performed At'),
            'align'     => 'left',
            'index'     => 'performed_at',
            'type'      => 'datetime',
            'width'     => '180',
            'sortable'  => false,
        ]);            
```
Or add an entry "Refreshed At" in the translation file.